rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -32,11 +32,13 @@
                 </or>
             </when>
             <perform>
-                <hint title="Java APIs for local file system" effort="1" category-id="optional">
-                    <message>Java APIs for local file system identified in the file.
-                    
-                    Local files in containers are not permanent storage and can disappear when an instance of the application terminates. Avoid accessing local filesystems from applications running in OpenShift environments, and consider using a database or cloud-based storage system instead.
+                <hint title="Java APIs for local file system" effort="1" category-id="openshift-potential">
```

Comment:
new category openshift-potential is defined in https://github.com/mrizzi/windup-rulesets/blob/fdbe0afaee2a60c0140b0f4418e4b148e09ab84e/rules-reviewed/core.windup.categories.xml#L43-L46